Parser/Client/Browser.php

```diff
@@ -90,6 +90,7 @@ class Browser extends AbstractClientParser
         'CN' => 'CoolNovo',
         'CO' => 'CometBird',
         'CB' => 'COS Browser',
+        'C3' => 'Chim Lạc',
```

Comment:
I don't think it's a good idea to use Vietnamese characters
```yml
Chim Lac
```
@sgiehl do you think we should replace "Chim Lạc" with "Chim Lac"?